src/nplinker/metabolomics/gnps/gnps_downloader.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+import httpx
+
+from .gnps_format import GNPSFormat
+from .gnps_format import gnps_format_from_task_id
+
+
+class GNPSDownloader:
+    GNPS_DATA_DOWNLOAD_URL = 'https://gnps.ucsd.edu/ProteoSAFe/DownloadResult?task={}&view=download_clustered_spectra'
+    GNPS_DATA_DOWNLOAD_URL_FBMN = 'https://gnps.ucsd.edu/ProteoSAFe/DownloadResult?task={}&view=download_cytoscape_data'
+
+    def __init__(self, task_id: str, download_root: Path):
```

Comment:
The lesson I learned is that path parameter should use type `str` as default and could support type `Path`, but don't only use type `Path` because it's not that convenient to convert `str` to `Path` if users already have string path at hand.

Also, it's better to return a path in `str` than in `Path`.